cp-quickstart/stop-docker.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Source library 
+. ../utils/helper.sh
+
+docker-compose down
```

Comment:
I think we want to add `-v` to cleanup volumes (present or future) used by this project.

Reply:
Ack and done